thruster_control/CMakeLists.txt

```diff
@@ -193,51 +79,26 @@ target_include_directories(${NODE_APP_NAME} PUBLIC
     pthread
  )
 
+roslint_cpp(
+  src/thruster_control.cpp
+  src/thruster_control_main.cpp
+  include/thruster_control/thruster_control.h
+)
+
+roslint_add_test()

```

Comment:
@patapufate same about putting lint tests into the `## Testing ##` block.

Reply:
Fixed